exercises/practice/square-root/.meta/src/SquareRoot.example.elm

```diff
@@ -0,0 +1,27 @@
+module SquareRoot exposing (squareRoot)
+
+
+squareRoot : Int -> Int
+squareRoot radicand =
+    case radicand of
+        1 ->
+            1
+
+        _ ->
+            let
+                improveGuess : Float -> Float
+                improveGuess guess =
+                    let
+                        nextGuess =
+                            (guess + toFloat radicand / guess) / 2
+                    in
+                    if abs (guess - nextGuess) < 0.0001 then
+                        nextGuess
+
+                    else
+                        improveGuess nextGuess
+            in
+            toFloat radicand
+                / 2
```

Comment:
For readability, I think this needs some parenthesis or an intermediate variable name.